contrib/go-pg/pg/example_test.go

```diff
@@ -0,0 +1,33 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package pg
+
+import (
+	"context"
+	"log"
+
+	pggotrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/go-pg/pg"
```

Comment:
In order to be able to import `"gopkg.in/DataDog/dd-trace-go.v1/contrib/go-pg/pg"` here, you need to change the package name (for just this file) to `pg_test` See how we do it in other integrations, for instance [garyburd/redigo](https://github.com/DataDog/dd-trace-go/blob/9495abe376f6c458ce6f5c5620d3b1ef17e65d25/contrib/garyburd/redigo/example_test.go#L6)

Also, you can run `go test` in the `contrib/go-pg/pg` directory to see these errors locally. That way you don't have to push here and wait for CI to fail. For example when I test the package locally, I can see the error:
```
$ pwd
~/dd-trace-go/contrib/go-pg/pg
$ go test
# gopkg.in/DataDog/dd-trace-go.v1/contrib/go-pg/pg
import cycle not allowed in test
package gopkg.in/DataDog/dd-trace-go.v1/contrib/go-pg/pg (test)
	imports gopkg.in/DataDog/dd-trace-go.v1/contrib/go-pg/pg
FAIL	gopkg.in/DataDog/dd-trace-go.v1/contrib/go-pg/pg [setup failed]
$ 
```

Reply:
Should be fixed, thank you for explanation, i'm quite new in GO.